src/components/Player/Pages/Peers/playerPeersColumns.jsx

```diff
@@ -40,3 +44,10 @@ export default playerId => [{
   displayFn: row => <TablePercent val={getPercentWin(row.against_win, row.against_games)} />,
   sortFn: row => row.against_win / row.against_games,
 }];
+
+export const playerPeersColumns = (playerId) => {
+  const columns = playerPeersOverviewColumns(playerId);
+  columns.splice(2, 0, ...matchesWith);
```

Comment:
I think we should avoid specified indices and explicitly declare the order (reusing named columns/sets of columns as needed.